tests/test_language_modeling_example.py

```diff
@@ -0,0 +1,92 @@
+import json
+import os
+import re
+import subprocess
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+import pytest
+
+from .test_examples import TIME_PERF_FACTOR
+
+
+prev_quant_model_name = None
+prev_quant_rank = 0
+
+if os.environ.get("GAUDI2_CI", "0") == "1":
+    # Gaudi2 CI baselines
+    MODELS_TO_TEST = {
+        "bf16_1x": [
+            ("google/gemma-2b-it", "mamamiya405/finred", "Habana/gpt2", True, 9.5, 31.5),
```

Comment:
Where do these baseline_train and baseline_eval numbers come from? Just observation of running? Do we have a feel for what the acceptable dispersion of results is? 

Just worried about false positives leading to alert fatigue here.

Reply:
yes baseline come from observation of running